staging/src/k8s.io/component-base/config/validation/validation.go

```diff
@@ -27,6 +27,10 @@ func ValidateClientConnectionConfiguration(cc *config.ClientConnectionConfigurat
 	if cc.Burst < 0 {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("burst"), cc.Burst, "must be non-negative"))
 	}
+
+	if cc.QPS > 0 && cc.Burst == 0 {
```

Comment:
I'm failing to see the need for this. We are already stating that Burst must be greater than 0 above.

Reply:
The above check only ensure that `burst` is `non-negative` (equal or greater than 0).